docs/references/account/create-mfa-authenticator.md

```diff
@@ -1 +1 @@
-Add an authenticator app to be used as an MFA factor. Verify the authenticator using the [verify authenticator](/docs/references/cloud/client-web/account#verifyAuthenticator) method.
\ No newline at end of file
+Add an authenticator app to be used as an MFA factor. Verify the authenticator using the [verify authenticator](/docs/references/cloud/client-web/account#verifyMfaAuthenticator) method.
```

Comment:
Isn't it updateMfaAuthenticator? https://appwrite.io/docs/references/1.5.x/client-web/account#updateMfaAuthenticator

Reply:
u g h yes you're right
```suggestion
Add an authenticator app to be used as an MFA factor. Verify the authenticator using the [verify authenticator](/docs/references/cloud/client-web/account#updateMfaAuthenticator) method.
```